<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bot connection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="host">
           <property name="placeholderText">
            <string notr="true">127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="port">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>42489</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Login:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="login"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_MultipleConnexion">
         <property name="title">
          <string>Multiple connexion</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="multipleConnexion">
            <property name="text">
             <string>Multiple connexion where %NUMBER% into login change</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Max diff connect - selected:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxDiffConnectedSelected">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_connectBySeconds">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Connect by seconds:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="connectBySeconds">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> connect/s</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Connexion count target:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="connexionCountTarget">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Proxy">
         <property name="title">
          <string>Proxy</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="proxy"/>
            </item>
            <item>
             <widget class="QSpinBox" name="proxyport">
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_Server">
      <property name="title">
       <string>Servers</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTreeWidget" name="serverList">
         <column>
          <property name="text">
           <string>Server</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Players</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="serverListSelect">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_char">
      <property name="title">
       <string>Character</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="autoCreateCharacter">
         <property name="text">
          <string>Autocreate if no one exists</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="characterList">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="characterSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="move">
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="randomText">
         <property name="text">
          <string>Random text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chatRandomReply">
         <property name="text">
          <string>Do global chat random reply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bugInDirection">
         <property name="text">
          <string>Do bug in direction</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Connexion informations</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="numberOfBotConnected">
         <property name="text">
          <string>Number of bot connected: ?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="numberOfSelectedCharacter">
         <property name="text">
          <string>Selected character: ?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLastReplyTime">
         <property name="text">
          <string>Last reply time: ?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelQueryList">
         <property name="text">
          <string>Running query: ? Query with worse time: ?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>connexionCountTarget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_connectBySeconds</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>connectBySeconds</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleConnexion</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxDiffConnectedSelected</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
